components/automate-deployment/pkg/converge/stop_mitigation.go

```diff
@@ -81,7 +81,7 @@ func (r *safeServiceShutdownRunner) RunIfRequired(t target.Target, pkg habpkg.Ve
 	if !required {
 		return nil
 	}
-	logrus.Info("Restart mitigation is required.")
+	logrus.Infof("Restart mitigation is required for service %s", pkg.Name())
```

Comment:
These two logging changes were the result of me watching the log output of the shutdown sequence and being a bit confused about the ordering I saw. These two log changes helped clarify the situation.